python_src/tests/performance_manager/test_performance_manager.py

```diff
@@ -166,6 +167,60 @@ def mock__get_pyarrow_table(
     yield
 
 
+def flat_table_check(db_manager: DatabaseManager, service_date: int) -> None:
```

Comment:
i'm still not in love with the testing we have here, but we can call this after we've loaded up the RDS and check that some things are working as expected.